src/cinder/gl/Fbo.cpp

```diff
@@ -305,7 +305,7 @@ void Fbo::initMultisamplingSettings( bool *useMsaa, bool *useCsaa, Format *forma
 }
 
 // Iterate the Format's requested attachments and create any we don't already have attachments for
-void Fbo::prepareAttachments( const Fbo::Format &format, bool multisampling )
+void Fbo::prepareAttachments( const Fbo::Format &format, bool /*multisampling*/ )
```

Comment:
@richardeakin: Probably. As it is not a private function I was a bit reluctant to change the interface. and to be honest I didn't really dig into the code or the documentation to find out, why the parameter was there in the first place. If I had to guess, I'd say it is a refactoring artifact (the only call site I could find was in the `init` function).